tfhe-zk-pok/src/proofs/pke.rs

```diff
@@ -1678,123 +1478,30 @@ mod tests {
     fn test_proof_compression() {
         let d = 2048;
         let k = 320;
-        let big_b = 1048576;
+        let b_i = 512;
```

Comment:
why the change ?

Reply:
I think it was a mistake in the previous version, since the other tests for pke v1 use `b_i = 512`